apps/files_sharing/src/components/NewFileRequestDialog.vue

```diff
@@ -0,0 +1,421 @@
+<!--
+  - SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcDialog can-close
+		class="file-request-dialog"
+		data-cy-file-request-dialog
+		:close-on-click-outside="false"
+		:name="currentStep !== STEP.LAST ? t('files_sharing', 'Create a file request') : t('files_sharing', 'File request created')"
+		size="normal"
+		@closing="onCancel">
+		<!-- Header -->
+		<NcNoteCard v-show="currentStep === STEP.FIRST" type="info" class="file-request-dialog__header">
+			<p id="file-request-dialog-description" class="file-request-dialog__description">
+				{{ t('files_sharing', 'Collect files from others even if they don\'t have an account.') }}
```

Comment:
I am not 100% sure about our guidelines here, but I would think we do not use abbreviations

```suggestion
				{{ t('files_sharing', 'Collect files from others even if they do not have an account.') }}
```

Reply:
That sentence came right from Jan. I'll adjust anyway, no big deal